zombienet/bulletin-westend-local.toml

```
@@ -0,0 +1,48 @@
+[settings]
+node_spawn_timeout = 240
+
+[relaychain]
+default_command = "{{POLKADOT_BINARY}}"
+default_args = ["-lparachain=debug,xcm=trace,runtime::bridge-hub=trace,runtime::bridge=trace,runtime::bridge-dispatch=trace,bridge=trace,runtime::bridge-messages=trace", "--blocks-pruning=128", "--state-pruning=128" ]
```

We have no bridge stuff on the relay :)
```suggestion
default_args = ["-lruntime=debug,xcm=trace"]
```